metrics/dropwizard-metrics/src/main/java/org/apache/iotdb/metrics/dropwizard/MetricName.java

```diff
@@ -84,7 +81,15 @@ public String toFlatString() {
 
   @Override
   public String toString() {
-    return "MetricName{" + "name='" + name + '\'' + ", tags=" + tags + '}';
+    return "MetricName{"
+        + "name='"
+        + name
+        + "'"
+        + ", metricLevel="
+        + metricLevel.name()
```

Comment:
If we use the constructor without MetricLevel,  I think here will throw a NPE

Reply:
Ok, I will check.

Reply:
This has been finished.